_drafts/2023-05-11-ota-for-embedded-linux-devices.md

```diff
@@ -0,0 +1,446 @@
+---
+title: "OTA for Embedded Linux Devices: A practical introduction"
+author: tsarlandie
+tags: [linux, ota, swupdate]
+---
+
+<!-- excerpt start -->
+
+A core belief of Memfault is that we can ship faster when we have efficient
+tools to detect, debug and fix issues in the field. Fixing issues on your
+computer is not enough: the fix must reach the end users!
```

Comment:
Second try


A core belief of the Memfault team is that companies can build and ship products faster and with hire quality when they have efficient
tools to detect, debug and fix issues in the field. This is why we highly recommend that development teams build out OTA and device monitoring *first*, and then start building out features and iterating. 

Maybe at the end: It's how I watched the firmware team at Pebble operate, and we moved *quickly*

Reply:
Reworked the intro with the help of hemingwayapp.com and your suggestions. Hope you will like the result.

Reply:
Love the new one. 🚢